pkg/certificate/cleanup.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"time"
```

Comment:
so, in the `Refactor overlap to use it ony at rotate` commit (btw typo in ony-->only)
we change from doing cleanp when notBefore+overlap to cleanup when noAfter?
and the overlap will only be used now to discover the rotation itself?

Reply:
Same idea as `renewBefore` here https://docs.cert-manager.io/en/release-0.8/reference/certificates.html.

Reply:
then why not call it `renewBefore`

Reply:
It will break the API but I supose is not important and it's better to understand it, I will put a commit in place to rename it.

Reply:
no problem to do it in a separate PR

Reply:
ack, thanks.